server/@types/bapv.d.ts

```diff
@@ -49,3 +49,7 @@ export type BookingConfirmed = {
   visitReference: string
   hasPhoneNumber: boolean
 }
+
+export type BookingCancelled = {
```

Comment:
Using an object here rather than just a simple boolean for the phone number because we will be expanding to have email and/or phone.